Google.Api.Gax.Grpc.IntegrationTests/ClientBuilderBaseTest.cs

```diff
@@ -282,27 +335,40 @@ public class SampleClientBuilder : ClientBuilderBase<CallInvoker>
         {
             public static string DefaultEndpoint { get; } = "default.nowhere.com";
             public static string[] DefaultScopes { get; } = new[] { "scope1", "scope2" };
-            public ChannelPool ChannelPool { get; } = new ChannelPool(DefaultScopes);
+            public virtual ChannelPool ChannelPool { get; } 
```

Comment:
No need for the virtual anymore.
```suggestion
            public ChannelPool ChannelPool { get; } 
```